### PR TITLE
fix for Sidebar anomaly in documentation and example page #1149

### PR DESCRIPTION
**Pull request by JithinAji**
Sidebar anomaly in documentation and example page

Issue: #1149

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -2,6 +2,10 @@ body {
   font-family: "Philosopher";
 }
 
+.sidebar .scrolling {
+  display: none;
```
Comment (shahednasser) [inline assets/css/main.css:6]: What's this line for?

Comment (JithinAji) [reply assets/css/main.css:6]: That I added while testing and forgot to remove afterwards. Can I submit another pull request after removing the same. 
Is there anything else that needs to be changed?

Comment (shahednasser) [reply assets/css/main.css:6]: you can just commit the changes to the same branch. No need to create a new PR.

Comment (JithinAji) [reply assets/css/main.css]: did the same. Please have a look.

**Comment by shahednasser** [comment]
Great work thanks!

@all-contributors please add @JithinAji for code and bug

**Comment by JithinAji** [comment]
Thank you tooo :)))

**Pull request merged**